edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/device/INestingInfo.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Device
+{
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public interface INestingInfo
+    {
+        void BindToParent(IIdentity parent);
+
+        Option<IIdentity> KnownParent { get; }
```

Comment:
Nit: Rename to Parent { get; }

Reply:
I used the term "known parent" because it is not necessarily the real parent - we just try to figure out from the subscription change. Sometimes what we set here as KnownParent is some great parent, also it is normal that the parent is not known (when it never subscribed) and I wanted to emphasize that Option.None is not the usual safeguard for nulls coming from programming errors, but a normal scenario